src/content/changelog/4-63-0-2020-04-15.md

```diff
@@ -0,0 +1,18 @@
+---
+title: '4.63.0: 2020-04-15'
+date: 2020-04-15T04:00:00.000Z
+version: 4.63.0
+changelog:
+  - API
+---
+
+### Changed
+- The [Linode Kubernetes Engine](https://www.linode.com/products/kubernetes/) (LKE) is out of Beta. All [LKE endpoints](/api/v4/lke-clusters) now point to the `/v4` server instead of the `/v4beta` server.
+  - **Note:** All `/v4` endpoints are available from the `/v4beta` server.
+
+- The View Kubernetes API Endpoint  ([GET /lke/clusters/{clusterId}/api-endpoint](/api/v4/lke-clusters-cluster-id-api-endpoint)) has been removed from the API in favor of the List Kubernetes API Endpoints ([GET /lke/clusters/{clusterId}/api-endpoints](/api/v4/lke-clusters-cluster-id-api-endpoints)) endpoint.
```

Comment:
good to go after removing the View Kubernetes API Endpoint link

Reply:
done! thank you!